src/Site.php

```diff
@@ -173,6 +173,39 @@ public function __construct($site_name_or_id = null)
         }
     }
 
+    /**
+     * Magic method dispatcher for site option fields, for convenience in Twig views.
+     *
+     * Called when explicitly invoking non-existent methods on the Site object. This method is not
+     * meant to be called directly.
+     *
+     * @example
+     * The following example will dynamically dispatch the magic __call() method with an argument
+     * of "users_can_register" #}
+     *
+     * ```twig
+     * {% if site.users_can_register %}
+     *   {# Show a notification and link to the register form #}
+     * {% endif %}
+     * @link https://secure.php.net/manual/en/language.oop5.overloading.php#object.call
+     * @link https://github.com/twigphp/Twig/issues/2
+     * @api
+     *
+     * @param string $option     The name of the method being called.
+     * @param array  $arguments Enumerated array containing the parameters passed to the function.
+     *                          Not used.
+     *
+     * @return mixed The value of the option field named `$field` if truthy, `false` otherwise.
+     */
+    public function __call($option, $arguments)
+    {
+        if (\method_exists($this, 'option') && $meta_value = $this->option($option)) {
```

Comment:
We should probably return the value without evaluating it?
```php
return $this->option($option);
```

Reply:
Ageed, fixed in https://github.com/timber/timber/pull/2798/commits/615917236bef098079189ab9fcf921b8e1eac0f9